source/interprocedural_analyses/taint/taintAnalysis.ml

```diff
@@ -681,10 +620,9 @@ let run_taint_analysis
   let override_graph_shared_memory_read_only =
     Interprocedural.OverrideGraph.SharedMemory.read_only override_graph_shared_memory
   in
-  Statistics.performance ~name:"Overrides computed" ~phase_name:"Computing overrides" ~timer ();
+  StepLogger.finish ~step_logger ~integers:[];
 
-  Log.info "Indexing global constants...";
-  let timer = Timer.start () in
+  let step_logger = StepLogger.start ~start_message:"indexing global constants..." ~end_message:"Indexed global constants" in
```

Comment:
You lost an uppercase here and you changed the end message. Again, the end message is important to preserve for us :/ 
```suggestion
  let step_logger = StepLogger.start ~start_message:"Indexing global constants" ~end_message:"Finished constant propagation analysis" in
```